templates/data_management/object_detail.html

```diff
@@ -44,7 +44,7 @@ <h3>{{ object.name }}</h3>
 
       {% if object.external_object %}
         <dt class="col-sm-3">External Object DOI or Unique Name</dt>
-        <dd class="col-sm-9">{{ object.external_object.doi_or_unique_name }}</dd>
+        <dd class="col-sm-9">{{ object.external_object.other_unique_name }}</dd>
 
```

Comment:
Not sure what this is doing, but should it mention `identifier`. Also title needs fixing.

Reply:
Resolved in https://github.com/FAIRDataPipeline/data-registry/commit/9e1fc00bf491a27c2f89ee1acf4a4453dec0b6d7